menu_state: 1
points: 0.1
true_false_state: true
rate: 3